### PR TITLE
moving vector-add to new repo.

### PR DESCRIPTION
**Pull request by Jenn500**
moving the latest vector-add to new repo, updated sample.json, added vector-add-usm.vcxproj
Signed-off-by: jennifer

### CODE REVIEW

**Comment by anjgola** [comment]
@mkitez  did CI run on this? @Jenn500  I am not sure if it did so let us wait until Roman confirms before merging it.

**Comment by Jenn500** [comment]
all checks are failed. But could not find the detail on why.

File: DirectProgramming/DPC++/DenseLinearAlgebra/vector-add/sample.json [hunk 1]
```diff
@@ -8,64 +9,16 @@
...
+      {"id": "cpu_usm","build": ["make build_usm"],"run": ["make run_usm"],"clean": ["make clean"]},
+      {"id": "fpga_emu_buffers","build": ["make clean -f Makefile.fpga","make fpga_emu -f Makefile.fpga"],"run": ["make run_emu -f Makefile.fpga"]},
+      {"id": "fpga_report_buffers","build": ["make report -f Makefile.fpga"],"clean": ["make clean -f Makefile.fpga"]}
+      ],
+	"windows":[
+	  {"id": "cpu_buffers","build": ["nmake -f Makefile.win"],"run": ["nmake -f Makefile.win run"],"clean": ["nmake -f Makefile.win clean"]},
+      {"id": "cpu_usm","build": ["nmake -f Makefile.win build_usm"],"run": ["nmake -f Makefile.win run_usm"],"clean": ["nmake -f Makefile.win clean"]},
+      {"id": "fpga_emu_buffers","build": ["nmake -f Makefile.win.fpga clean","nmake -f Makefile.win.fpga"],"run": ["nmake -f Makefile.win.fpga run"]}
+	  ]
+    }
```
Comment (mkitez) [inline DirectProgramming/DPC++/DenseLinearAlgebra/vector-add/sample.json:23]: The existing configurations are the correct ones in terms of syntax. Please, review them and make required updated if needed.

Comment (Jenn500) [reply DirectProgramming/DPC++/DenseLinearAlgebra/vector-add/sample.json:23]: I'll make the change and resubmit.

Comment (sravanikonda) [reply DirectProgramming/DPC++/DenseLinearAlgebra/vector-add/sample.json:23]: @mkitez Jennifer is on vacation, can you help me understand what the required updates are?

Comment (anjgola) [reply DirectProgramming/DPC++/DenseLinearAlgebra/vector-add/sample.json:23]: @sravanikonda  The json field's value in this PR is outdated. It is not using the new format specified here : https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object

**Comment by Jenn500** [comment]
I've updated the sample.json. please check if it's correct.

**Pull request merged**